releasenotes-builder/src/main/resources/com/github/checkstyle/templates/xdoc_freemarker.template

```diff
@@ -5,7 +5,7 @@
         <ul>
         <#list breakingMessages as message>
           <li>
-            ${message.title}. Author: ${message.author}<#if message.issueNo != -1> <a href="https://github.com/checkstyle/checkstyle/issues/${message.issueNo}">#${message.issueNo}</a></#if>
+            ${message.title}. Author: ${message.author}<#if message.issueNo != -1> <a href="https://github.com/${remoteRepoPath}/issues/${message.issueNo}">#${message.issueNo}</a></#if>
```

Comment:
in next PR, please add "\n" here make line about 100 symbols and be ease to review
it will also resolve our problem with generation of long lines in result XML.

Reply:
yes, this is a separate issue, so I am waiting for that to do it.
I did this for sevntu since it is a new template,